### PR TITLE
Filters menu standardising

### PR DESCRIPTION
**Pull request by mikeallbutt**
Screen Shot 2022-03-28 at 13 12 54
Screen Shot 2022-03-28 at 13 13 05

### CODE REVIEW

**Comment by corsacca** [comment]
@mikeallbutt why is default filters better than "all" ?

**Comment by mikeallbutt** [comment]
There are lots of uses of 'all' in the lists and this brings better clarity to what the all is.

File: archive-template.php [hunk 1]
```diff
@@ -192,7 +192,7 @@ class="second-bar show-for-small-only center list-actions-bar"><!--  /* MOBILE V
                     <div class="section-body">
                         <ul class="accordion" id="list-filter-tabs" data-responsive-accordion-tabs="accordion medium-tabs large-accordion"></ul>
                         <div style="margin-bottom: 5px">
-                            <a data-open="filter-modal"><img class="dt-blue-icon dt-icon" style="display: inline-block; margin-right:12px" src="<?php echo esc_html( get_template_directory_uri() . '/dt-assets/images/circle-add.svg' ) ?>"/><?php esc_html_e( "Add new filter", 'disciple_tools' ) ?></a>
+                            <a data-open="filter-modal"><img class="dt-blue-icon dt-icon" style="display: inline-block; margin-right:12px" src="<?php echo esc_html( get_template_directory_uri() . '/dt-assets/images/circle-add.svg' ) ?>"/><?php esc_html_e( "Create custom filter", 'disciple_tools' ) ?></a>
```
[the remark this answers is not in the document]
Comment (mikeallbutt) [reply archive-template.php:195]: @corsacca thanks for asking. 

When I was using the link and thinking about it, I realised for me that I was thinking that the `Add new filter` was not actually what was happening. I was creating a filter (hence the suggested change to `Create custom filter`) and then if I wanted to add it (to the list) I would then save it.

Also, by including the word `custom` it tells the user where the filter will appear once created. Without the `custom` the new user (or someone who hasn't used this action before) might think they can add a new filter to the other filter menus. Years ago when I first started with DT, that's what I thought too.

Yes this text has been `add new filter` for a long time but I was more thinking about the new people who will be joining in the future and think this wording of `create custom filter` better explains what the link does. It is not a huge issue to have this new string to be translated IMO.

Screen Shot 2022-03-29 at 07 31 57

**Comment by corsacca** [comment]
Thank you @mikeallbutt!

**Pull request merged**